src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -151,6 +154,66 @@ public void testWriteStringData() {
     }
 
     /**
+     * Test writing to the database using string protocol through UDP.
+     */
+    @Test
+    public void testWriteStringDataThroughUDP() {
```

Comment:
there is some issue for test case through the udp function is right. I will figure out what
's the root cause.